qlib/contrib/model/pytorch_transformer.py

```diff
@@ -0,0 +1,312 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+
+from __future__ import division
+from __future__ import print_function
+
+import os
+import numpy as np
+import pandas as pd
+import copy
+import math
+from ...utils import get_or_create_path
+from ...log import get_module_logger
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.utils.data import DataLoader
+
+from .pytorch_utils import count_parameters
+from ...model.base import Model
+from ...data.dataset import DatasetH, TSDatasetH
+from ...data.dataset.handler import DataHandlerLP
+
+import pdb
+
+# qrun benchmarks/Transformer/workflow_config_transformer_Alpha158.yaml

```

Comment:
where is `benchmarks/Transformer/workflow_config_transformer_Alpha158.yaml`?

Reply:
It is now in the "benchmark" path.